src/index.html

```diff
@@ -1,13 +1,57 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="iMac"
+        class="image card__image"
+      >
+      <div class="content-block card__content-block">
+        <div class="info content-block__item-info">
+          <span class="name info__name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</span>
+          <span class="code">Код товара: 195434</span>
```

Comment:
For texts and titles use p and h tags, also fix your BEM. 
```suggestion
          <h2 class="card__title">
            Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
          </h2>
          <p class="card__code">
            Код товара: 195434
          </p>
```